builder/azure/common/inspector.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package common
+
+import (
```

Comment:
Is this common file meant to replace the other two inspectors? 

I see the code added to each builder but wonder if they can be consolidated.

Reply:
Oh, nevermind this, I had tried to reify the inspector implementation in common, but since the private functions like `handleBody` are used in different places in each builder, I opted no to do this right away.
I'll remove this from the PR, we can revisit later.